docker-compose/README.md

```diff
@@ -98,13 +98,25 @@ Build and run sdk test suite containers with the command below. Depending on whi
 Build and run the test runner to start integration tests execution: \
 ```docker-compose -f docker-compose.integration_tests_runner.yml --env-file .env-test up --build```
 
+There are utility scripts in .devcontainer/integration-tests:
+* startPlatform.sh: builds and starts the mqtt broker and the platform
```

Comment:
Improvement idea:
- "startPlatform" indicates that it's starting something in the background and you need a "stopPlatform" script to stop it.
- "startIntegrationTest" - the description indicates that this script will exit when it's done, then a better name could be "runIntegrationTests".

startPlatform()
runIntegrationTests()
stopPlatform()